InteractionBase/src/vis/ViewSwitcherMenu.cpp

```diff
@@ -87,7 +87,10 @@ void ViewSwitcherMenu::endFocusMode(Visualization::Item *target)
 		QPoint pos = indexOf(entry);
 		auto nameAfter = entry->nameField()->text();
 		if (scene()->viewItems()->viewItem(nameAfter) == nullptr && nameAfter != nameBefore_)
-			scene()->viewItems()->newViewItem(nameAfter, pos);
+		{
+			if (scene()->viewItems()->loadView(nameAfter, pos)) {}
```

Comment:
Hmm I thought I commented on this one, but I don't see my comment. Use `if (! ...) ... //without else`